.github/verible.waiver

```diff
@@ -15,3 +15,5 @@ waive --rule=interface-name-style --location="hw/ip/reqrsp_interface/src/reqrsp_
 waive --rule=interface-name-style --location="hw/ip/tcdm_interface/src/tcdm_interface.sv"
 # We have some long lines
 waive --rule=line-length --location="hw/*"
+# Some files are replaced by Bender thus do not respect the module-filename rule
```

Comment:
What does this mean? Are you sure Verilator is then using the correct `hw/ip/spatz/src/vregfile_fpga.sv` from the Spatz repo after compilation? Just not sure if this replacement is due to another module with the same filename in another bender dependency.

Reply:
This is a waiver for verible, the formatted from google, not verilator. The rationale is that we have some files where the filename does not align with the module name, like in all cases where we need to swap module with a bender target (tech cells etc), so we waive the check of this rule in verible

verilator is safe since we do not use the `-fpga` target there

Reply:
Oh yeah sorry I thought it was the waiver for Verilator, looks good.